template.yml

```diff
@@ -277,6 +277,14 @@ Resources:
               - - '{{resolve:ssm:'
                 - !Sub "${EnvironmentName}SentryDsn"
                 - ':1}}'
+        - Namespace: aws:elasticbeanstalk:application:environment
```

Comment:
linebreak is missing here, keep the original formating

Reply:
done